src/CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ project(qtimgui)
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOUIC ON)
-find_package(Qt5 COMPONENTS Core Quick Gui Widgets REQUIRED)
+
+# We're using qt versionless targets so our minimal Qt version is 5.15, we support Qt6 as well
+# See https://www.qt.io/blog/versionless-cmake-targets-qt-5.15
+find_package(Qt6 COMPONENTS COMPONENTS  Core Quick Gui Widgets OpenGLWidgets )
+if (NOT Qt6_FOUND)
+    find_package(Qt5 5.15 REQUIRED COMPONENTS Core Quick Gui Widgets)
+    if (NOT Qt5_FOUND)
+        message(FATAL_ERROR "We only support  only Qt 5.15 and higher.")
+    endif()
+endif()
+
+message(STATUS  Qt ${QT_DEFAULT_MAJOR_VERSION})
```

Comment:
This message should be a bit clearer to the user. Like "Compiling with Qt X". I saw it, and was just a bit puzzled.